kube-janitor/Chart.yaml

```diff
@@ -4,9 +4,9 @@ description: A Helm chart for running kube-janitor
 name: kube-janitor
 version: 0.1.0
 maintainers:
-  - name: Robert Sheehy
-    email: gameboy1092@gmail.com
-  - name: Kevin Simons
-    email: simkev2@gmail.com
+- name: Robert Sheehy
```

Comment:
Also while you are in here would you mind changing "Robert Sheehy" to "theMagicalKarp" and "Kevin Simons" to "SimKev2"? The helm repo requires the name maps to a github user. I had made the change in helm, but had never propagated it here. 
https://github.com/helm/charts/blob/master/incubator/kube-janitor/Chart.yaml#L8

Reply:
Will do.